### PR TITLE
[WIP] revise modal margin

### PR DESCRIPTION
**Pull request by uniqueeest**
As I checked the modals, there were no parts that needed to be corrected except for the two modals found in the issue. So, I just proceeded to revise the two modals to the requested margin.

### CODE REVIEW

**Comment by joel-jeremy** [comment]
@uniqueeest Please add a release note. You can refer to https://actualbudget.org/docs/contributing/#writing-good-release-notes.

**Comment by uniqueeest** [comment]
@joel-jeremy I already add a release note. Is that how you write it?

**Comment by uniqueeest** [comment]
Thank you @youngcw !

File: packages/desktop-client/src/components/modals/CreateEncryptionKeyModal.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ export function CreateEncryptionKeyModal({
     <Modal
       {...modalProps}
       style={{
-        padding: '0 10px',
+        padding: 10,
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/CreateEncryptionKeyModal.tsx:68]: I think Modal has a default padding so we can remove this one.

File: packages/desktop-client/src/components/modals/FixEncryptionKeyModal.tsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ export function FixEncryptionKeyModal({
         hasExistingKey ? 'Unable to decrypt file' : 'This file is encrypted'
       }
       style={{
-        padding: '0 10px',
+        padding: 10,
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/modals/FixEncryptionKeyModal.tsx:64]: Same here

Comment (uniqueeest) [reply packages/desktop-client/src/components/modals/FixEncryptionKeyModal.tsx]: @joel-jeremy Corrections are complete.

**Review by joel-jeremy — APPROVED** [review]
LGTM!

**Pull request merged**